tensorflow/compiler/xla/literal_util.cc

```diff
@@ -917,6 +933,16 @@ LiteralUtil::GetMutableArraySlice(Literal* literal) {
 }
 
 template <>
+/* static */ tensorflow::gtl::MutableArraySlice<half>
```

Comment:
it's only a problem if the literals are sent over the network of course.  Is that ever true?